src/main/java/com/exasol/adapter/dialects/hive/HiveColumnMetadataReader.java

```diff
@@ -30,8 +30,11 @@ public HiveColumnMetadataReader(final Connection connection, final AdapterProper
 
     @Override
     public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
-        if (jdbcTypeDescription.getJdbcType() == Types.DECIMAL) {
+        final int jdbcType = jdbcTypeDescription.getJdbcType();
+        if (jdbcType == Types.DECIMAL) {
             return mapDecimal(jdbcTypeDescription);
+        } else if (jdbcType == Types.BINARY) {
+            return DataType.createMaximumSizeVarChar(DataType.ExaCharset.UTF8);
```

Comment:
I am surprised that this works. How does binary data fit into a UTF-8 column?

Reply:
I have no idea, this was in the dialect before, I suppose. I just removed it from the base class